desktop/extensions-sdk/extensions/publish.md

```diff
@@ -30,9 +33,46 @@ These guidelines don't replace Docker's terms of service or guarantee approval.
 - Test your extension on various platforms (Mac, Windows, Linux)
 - Read the [Terms of Service](https://www.docker.com/legal/extensions_marketplace_developer_agreement/){: target="_blank" rel="noopener" class="_" }
 
-### After you submit
+### Which process to choose
+
+When creating the issue on the extensions submissions repository, you can choose to request a review from the Docker
+Extension team before listing your extension. Whether you choose one or the other, the publishing process will be
+different.
+
+#### No review requested
```

Comment:
should we mention that a published not reviewed extension can be reviewed later?